files/Aboot/boot0.j2

```diff
@@ -395,6 +395,9 @@ write_boot_configs() {
         fi
     fi
 
+    # setting secure_boot_enable=true when secure boot enabled
+    [ -f /bin/securebootctl ] && securebootctl secureboot -display | grep -i "Secure Boot enable" -q && echo "secure_boot_enable=true" >> /tmp/append
```

Comment:
>securebootctl secureboot -display | grep -i "Secure Boot enable" -q [](start = 33, length = 67)

Is it possible to test secure boot enabled without grep?

Reply:
I will ask Arista about it.

Reply:
The arista secure boot using the similar way.